cms/templates/admin/cms/page/change_form.html

```diff
@@ -16,11 +16,11 @@
 {{ block.super }}
 {# load nonconflict pre and post files to fix conflict if there is another jQuery loaded #}
 {# the code is included by a external file because the code has to be loaded into a python file where the wizard is loaded as well #}
-<script src="{% static "cms/js/modules/jquery.noconflict.pre.js" %}" type="text/javascript"></script>
-<script src="{% static "admin/js/urlify.js" %}" type="text/javascript"></script>
-<script src="{% static "cms/js/dist/bundle.admin.base.min.js" %}" type="text/javascript"></script>
-<script src="{% static "cms/js/dist/bundle.admin.changeform.min.js" %}" type="text/javascript"></script>
-<script src="{% static "cms/js/modules/jquery.noconflict.post.js" %}" type="text/javascript"></script>
+<script src="{% static "cms/js/modules/jquery.noconflict.pre.js" %}?{% get_cms_version %}" type="text/javascript"></script>
```

Comment:
this should be a template tag that wraps ``{% static %}`` maybe called ``static_with_version``.